### PR TITLE
Makes the description of some TC items provides more information about the item

### PR DESCRIPTION
**Pull request by ma44**

### CODE REVIEW

**Comment by ma44** [comment]
Updates soup to be more grammar friendly (thanks lexorion)

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -557,8 +557,9 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 
 /datum/uplink_item/stealthy_weapons/soap
 	name = "Syndicate Soap"
-	desc = "A sinister-looking surfactant used to clean blood stains to hide murders and prevent DNA analysis. \
-			You can also drop it underfoot to slip people."
+	desc = "A sinister-looking surfactant that cleans up clean blood stains and remove DNA evidence much more faser than regular soup.\
```
Comment (Lexorion) [inline code/modules/uplink/uplink_item.dm]: Should be soap,not soup.

Comment (TechnoAlchemisto) [inline code/modules/uplink/uplink_item.dm]: "and remove DNA evidence much more faser than regular soup."

Is this a joke?

**Comment by AdamElTablawy** [comment]
can you just like

die or something

every pr has been ass so far

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -518,7 +518,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 /datum/uplink_item/stealthy_weapons/martialarts
 	name = "Martial Arts Scroll"
 	desc = "This scroll contains the secrets of an ancient martial arts technique. You will master unarmed combat, \
-			deflecting all ranged weapon fire, but you also refuse to use dishonorable ranged weaponry."
+			deflecting all ranged weapon fire and learning several fighting techniques, but you also refuse to use dishonorable ranged weaponry."
```
Comment (ike709) [inline code/modules/uplink/uplink_item.dm]: This addition is already implied by the first sentence and first part of the second sentence, as well as by the name of the item.

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (ike709) [inline code/modules/uplink/uplink_item.dm]: Everything about this is bad.

File: code/modules/uplink/uplink_item.dm [hunk 3]
```diff
@@ -797,7 +798,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 
 /datum/uplink_item/device_tools/military_belt
 	name = "Military Belt"
-	desc = "A robust seven-slot red belt that is capable of holding all manner of tatical equipment."
+	desc = "A robust seven-slot red belt that is capable of holding all manner of tatical equipment. Does not contain anything"
```
Comment (ike709) [inline code/modules/uplink/uplink_item.dm]: "capable of holding" implies that it _can_ but _doesn't_ already.

Literally the only thing to validate this PR's existence is correcting the "tatical" typo, but you didn't even fix the typo.

**Comment by ike709** [comment]
You should be jobbanned from description editing.

**Comment by ma44** [comment]
Cause i should be jobbanned from description editing and kill myself

**Pull request closed without merging**